FastSimulation/Tracking/python/IterativePixelLessStep_cff.py

```diff
@@ -33,6 +33,8 @@
 #                                     'TEC1_pos+TEC2_pos','TEC2_pos+TEC3_pos','TEC3_pos+TEC4_pos','TEC3_pos+TEC5_pos','TEC4_pos+TEC5_pos',
 #                                     'TID1_neg+TID2_neg','TID2_neg+TID3_neg',
 #                                     'TEC1_neg+TEC2_neg','TEC2_neg+TEC3_neg','TEC3_neg+TEC4_neg','TEC3_neg+TEC5_neg','TEC4_neg+TEC5_neg']
+
+from RecoLocalTracker.SiStripClusterizer.SiStripClusterChargeCut_cfi import *
```

Comment:
I think what happens is the following
- from the import below it follows that a `process` loading this configuration will have `process.pixelLessStepSeedLayers` member, that has `refToPSet_`s to `SiStripClusterChargeCutTight`
- all `refToPSet_`s are resolved when the `process` is "transformed" to C++ ParameterSets (`cms.Process.fillProcessDesc()`)
- if the import above is missing, the `process` does not have the `SiStripClusterChargeCutTight` member referenced by a `refToPSet_` in `process.pixelLessStepSeedLayers`

I have one idea to circumvent this "behind the scenes" dependence, but I want to test it first myself.